homeassistant/components/daikin/__init__.py

```diff
@@ -28,11 +28,18 @@
 COMPONENT_TYPES = ["climate", "sensor", "switch"]
 
 CONFIG_SCHEMA = vol.Schema(
-    {
-        DOMAIN: vol.Schema(
-            {vol.Optional(CONF_HOSTS, default=[]): vol.All(cv.ensure_list, [cv.string])}
-        )
-    },
+    vol.All(
+        cv.deprecated(DOMAIN, invalidation_version="0.112.0"),
```

Comment:
Please do two releases in the future from the release that published the change, ie 0.113.0.